src/main/java/com/example/jdbcdemo/configuration/JdbcConfig.java

```diff
@@ -0,0 +1,166 @@
+package com.example.jdbcdemo.configuration;
+
+
+import com.example.jdbcdemo.model.Match;
+import com.example.jdbcdemo.model.Team;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class JdbcConfig {
+    static String URL = "jdbc:postgresql://localhost:5432/tournament";
+    static String USERNAME = "postgres";
+    static String PASSWORD = "postgres";
+
+    static Connection connection;
+
+    public void saveTeam(List<Team> teams) {
+        try {
+            connection = DriverManager.getConnection(URL, USERNAME, PASSWORD);
+            PreparedStatement preparedStatement = connection.prepareStatement("INSERT INTO TEAMS VALUES(?, ?, ?, ?)");
+            for (int i = 0; i < teams.size(); i++) {
+                preparedStatement.setInt(1, i + 1);
+                preparedStatement.setString(2, teams.get(i).getName());
+                preparedStatement.setString(3, teams.get(i).getCapitan());
+                preparedStatement.setString(4, teams.get(i).getCoach());
+                preparedStatement.executeUpdate();
+            }
+        } catch (SQLException throwables) {
```

Comment:
You can name it as you wish but I recommend to name it `ex` / `exception`.
This terminology is simplier than `throwable`.